common/src/abstractions/platformUtils.service.ts

```diff
@@ -28,7 +29,8 @@ export abstract class PlatformUtilsService {
     readFromClipboard: (options?: any) => Promise<string>;
     supportsBiometric: () => Promise<boolean>;
     authenticateBiometric: () => Promise<boolean>;
-    getDefaultSystemTheme: () => Promise<'light' | 'dark'>;
-    onDefaultSystemThemeChange: (callback: ((theme: 'light' | 'dark') => unknown)) => unknown;
+    getDefaultSystemTheme: () => Promise<ThemeType.Light | ThemeType.Dark>;
+    onDefaultSystemThemeChange: (callback: ((theme: ThemeType.Light | ThemeType.Dark) => unknown)) => unknown;
+    getEffectiveTheme: () => Promise<ThemeType>;
```

Comment:
We'll need to implement this for the `electronPlatformUtils` and `CliPlatformUtils` in the jslib project.

Reply:
Good catch. Done